src/content/docs/en/guides/integrations-guide/alpinejs.mdx

```diff
@@ -96,31 +96,45 @@ Once the integration is installed, you can use [Alpine.js](https://alpinejs.dev/
 
 Check our [Astro Integration Documentation][astro-integration] for more on integrations.
 
-## Limitations
+## Options
 
-The Alpine.js integration does not give you control over how the script is loaded or initialized. If you require this control, consider [installing and using Alpine.js manually](https://alpinejs.dev/essentials/installation). Astro supports all officially documented Alpine.js manual setup instructions, using `<script>` tags inside of an Astro component.
+### entrypoint
```

Comment:
```suggestion
### `entrypoint`
```

Reply:
I checked and it looks like no integration docs is using backticks in headlines. Should we stay consistent?

Reply:
There are a couple of examples, here you have one. https://github.com/withastro/docs/blob/a5bc295a807b1493b34d68d19f26eda18b6b165a/src/content/docs/en/guides/integrations-guide/mdx.mdx#L136
I was thinking as well about adding the the type and the def type like in that example, let me know what you think.

Reply:
I don't mind tbh 👍